bemuse/src/custom-folder/index.ts

```diff
@@ -283,6 +287,50 @@ async function updateFolders(
   }
 }
 
+async function removeFolders(
+  state: CustomFolderState,
+  handle: FileSystemDirectoryHandle,
+  io: CustomFolderScanIO
+): Promise<ScanIterationResult | undefined> {
+  const { log, setStatus } = io
+
+  if ((state.foldersToRemove?.length ?? 0) > 0) {
+    let remainingSongs
```

Comment:
Don’t forget to update this one krub:

```suggestion
    let remainingSongs = state.songs || []
```